#! /bin/sh /usr/share/dpatch/dpatch-run
## network-bridge.dpatch by Bastian Blank <waldi@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Load netloop module

@DPATCH@
diff -urNad xen-common~/tools/examples/network-bridge xen-common/tools/examples/network-bridge
--- xen-common~/tools/examples/network-bridge	2006-07-05 18:34:34.000000000 +0000
+++ xen-common/tools/examples/network-bridge	2006-09-27 07:58:21.092674561 +0000
@@ -59,6 +59,8 @@
 findCommand "$@"
 evalVariables "$@"
 
+modprobe netloop > /dev/null 2>&1 || true
+
 is_network_root () {
     local rootfs=$(awk '{ if ($1 !~ /^[ \t]*#/ && $2 == "/") { print $3; }}' /etc/mtab)
     local rootopts=$(awk '{ if ($1 !~ /^[ \t]*#/ && $2 == "/") { print $4; }}' /etc/mtab)
